applications/aihorde-application.md

```diff
@@ -0,0 +1,202 @@
+support my game development. I have a community in the thousands, with dozens
+of third party web services relying on the uptime of the AI Horde to serve their
+own communities! While I have a few volunteers helping me with moderation and
+development, we have no formal governance or other structure. 
+
+**Governance**: I started developing AI Horde on my own, as a side project,
+and I’m still working on it only in my spare time. Currently I am a de-facto
+“benevolent dictator”, which is not a dynamic I enjoy, and any conflicts usually
+require me to step in. As far as governance is concerned, I can at best react.
+I just don’t have the mental bandwidth to build anything, nevermind the knowledge.
+For the record, I am running my community according to anarchist principles.
+
+**Infrastructure**: I’ve built the AI Horde to be able to be handled by one
+person on a shoestring budget. Fortunately I’m an infrastructure automation
+expert so that allowed me to scale to serve millions of HTTP requests per day,
+in a budget of less than 200$ per month. However I’m slowly reaching the limits
+of my own abilities to scale efficiently. For example, while I self-taught myself
+enough DBA skills to set up and maintain a postgresql DB, I have very little
+capacity to harden it as a cluster and ensure best practices. I could really use
+specialists in areas that I am myself not an expert in, such as Database
+administration, HAproxy etc. I could also use guidance on how to onboard other
+SysAdmins, due to the potential risk of other people having access to the servers.
+
+**Optimization**: Likewise, I do not have enough advanced programming knowledge
+(I’m self-taught and never programmed professionally) to fine-tune the code to a
+great degree. While I do make improvements on DB calls as I learn and improve myself,
+I can only do so much in my spare time. As such, I still have issues with things
+like race-conditions, or handling heavy operations. I could use people who can help
+or guide on these matters in python.
+
+**Funding**: I am still developing this service in my own spare time. This is slowly
+becoming more and more untenable as the popularity of the AI Horde increases. Currently
+I am doing all the below:
+
+* middleware development
+* backend development
+* frontend and bot development
+* infrastructure deployment and automation
+* public relations
+* community management
+* security and countermeasures research
+
+I am spread beyond thin and just have [one patreon](https://www.patreon.com/db0/) to help with my infrastructure costs.
+
+Ultimately, I would like the AI Horde to be a publicly funded project, where I can work
+on it full-time. 
+
+I have no idea how to achieve this and the challenges I face with even simple attempts
+at financing myself have left me permanently stressed out.
+
+**Trademarks**: I have no trademarks or knowledge on how to handle this sort of thing,
+which sooner or later I think I need to think about 
```

Comment:
This is something we can help with. Trademarks can be a pain to deal with, however its a great way to keep corporations from stealing your work.